docs/core/profiles.md

```diff
@@ -150,11 +150,19 @@ speed_factor:
   road_environment: {tunnel: 1.2}
```

Comment:
Should we mention that this is a special notation to avoid indentation in yaml?

```
speed_factor:
   road_class:
     motorway: 2
   road_environment
     tunnel: 1.2
```